PythonRegression/tests/features/machine4/4_stitching.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Ensure node reliability while stitching a side tangle 
+	A stitching tangle transaction needs to be issued to the tangle referencing
+	a transaction from the main tangle, and a transaction from the side tangle
+	and check consistency needs to be called to ensure the transaction does not 
+	crash the node. If the node responds to the api call, it has not crashed. A
+	transaction will then need to be cast referencing the stitching transaction
+	should also not crash the node. 
+	
+	Scenario: Check consistency on a stitching transaction responds
+		
+		Given a stitching transaction is issued on "nodeA" with the tag "STITCHING"
+		And check_consistency is called on this transaction
+		Then the response should return "False"	
+		
+	Scenario: A transaction referencing the stitch should not crash the node
```

Comment:
1. I think instead of having multiple scenarios that depend on one another is not something we should prefer to do (I may have said otherwise before but now I am thinking about this again). Imagine this scenario fails in the CI. Later the dev will want to recreate the error by running this scenario with the `@now` tag.
He won't understand that the scenarios are dependent... Maybe merge it into 1 secenario?
2. Call checkConsistency again?